src/azure-cli/azure/cli/command_modules/role/commands.py

```diff
@@ -209,10 +203,3 @@ def load_command_table(self, _):
         g.command('add', 'add_member')
         g.command('remove', 'remove_member')
         g.custom_command('check', 'check_group_membership', client_factory=get_graph_client_groups)
-
-    with self.command_group('identity', identity_sdk, min_api='2017-12-01') as g:
```

Comment:
`azure-mgmt-msi==0.2.0` doesn't support multi-API at all. See https://github.com/Azure/azure-sdk-for-python/tree/azure-mgmt-msi_0.2.0/azure-mgmt-msi

This `min_api='2017-12-01'` works because it is checked against `ResourceType.MGMT_AUTHORIZATION` specified for the whole `role` module:

https://github.com/Azure/azure-cli/blob/9c4b39efaea4a94a4f8187e9562e453962adc214/src/azure-cli/azure/cli/command_modules/role/__init__.py#L17-L19

This is certainly wrong.

After removing `min_api='2017-12-01'`, `az identity` will work in `xxx-hybrid` profiles.